pages/common/clockwork-cli.md

```diff
@@ -0,0 +1,15 @@
+# clockwork-cli
+
+> A command line interface for the Clockwork PHP debugging framework.
+
+- Monitor Clockwork logs for the current project:
```

Comment:
Monitor them for what?

Reply:
Monitor them for requests, it basically allows you to debug HTTP requests, etc. for Laravel.
![](https://raw.githubusercontent.com/ptrofimov/clockwork-cli/master/screenshots/clockwork-cli-trace.png)

Reply:
Ah, cool! I see.

(And fellow Ubuntu user! Hi!)

Reply:
Not my screenshot, but I do use Ubuntu 18.10. :stuck_out_tongue: